polkadot/xcm/pallet-xcm-benchmarks/src/generic/mod.rs

```diff
@@ -97,6 +98,14 @@ pub mod pallet {
 		/// If set to `Err`, benchmarks which rely on a universal alias will be skipped.
 		fn alias_origin() -> Result<(Location, Location), BenchmarkError>;
 
+		/// An XCM used to exercise the barrier logic in benchmarks.
+		///
+		/// This represents a call used to benchmark the cost of barrier evaluation in rejection
+		/// cases.
+		///
+		/// If set to `Err`, benchmarks which rely on a barrier check will be skipped.
+		fn worst_case_for_not_passing_barrier() -> Result<Xcm<Instruction<Self>>, BenchmarkError>;
```

Comment:
```suggestion
		fn worst_case_xcm_not_passing_barrier() -> Result<Xcm<Instruction<Self>>, BenchmarkError>;
```

Something like that reads better, what do you think?